pkg/ddevapp/global_dotddev_assets/commands/web/craft

```diff
@@ -3,11 +3,15 @@
 #ddev-generated
 ## Description: Run Craft CMS command inside the web container
 ## Usage: craft [flags] [args]
-## Example: "ddev craft db/backup" or "craft db/backup ./my-backups" (see https://craftcms.com/docs/4.x/console-commands.html)
-## ProjectTypes: craftcms
+## Example: "ddev craft db/backup" or "ddev craft db/backup ./my-backups" (see https://craftcms.com/docs/4.x/console-commands.html)
+## ProjectTypes: craftcms,php
 ## ExecRaw: true
 
-CRAFT_CMD_ROOT=${CRAFT_CMD_ROOT:="./"}
-
-cd "${CRAFT_CMD_ROOT}"
-php ./craft "$@"
+if grep -q "type: php" ./.ddev/config.yaml; then
```

Comment:
```suggestion
if [ "${DDEV_PROJECT_TYPE}" != "craftcms" ]; then
```

Reply:
I didn't manually test this, so please give it a quick manual review, but think it's better.